lib/src/base-styles/_nx-read-only.scss

```diff
@@ -23,8 +23,10 @@
 .nx-read-only__label {
   @include bold();
   margin: var(--nx-spacing-8x) 0 var(--nx-spacing-4x) 0;
+  max-width: var(--nx-width-text-wrapping);
 }
 
 .nx-read-only__data {
   margin: 0 0 var(--nx-spacing-2x) 0;
+  max-width: var(--nx-width-text-wrapping);
```

Comment:
I suspect we don't want this set for `.nx-read-only__data` that are children of `.nx-read-only--grid`.  The grid should be in charge of the layout there.

Reply:
> I suspect we don't want this set for `.nx-read-only__data` that are children of `.nx-read-only--grid`. The grid should be in charge of the layout there.

https://github.com/sonatype/sonatype-react-shared-components/pull/432/commits/471e9c58d35433af6175f3ce8ef936f8c7149b7a